tutorials/algorithms/04_vqe_advanced.ipynb

```diff
@@ -184,37 +196,41 @@
-      "                              Parameter(θ[4]): -2.5983258956331645,\n",
-      "                              Parameter(θ[5]): 1.568326000491598,\n",
-      "                              Parameter(θ[6]): -4.717618128585369,\n",
-      "                              Parameter(θ[7]): 0.3602072910298268},\n",
-      "    'optimal_point': array([ 4.29652046,  4.42696214,  0.54707542,  6.09294784, -2.5983259 ,\n",
-      "        1.568326  , -4.71761813,  0.36020729]),\n",
-      "    'optimal_value': -1.8572750175597519,\n",
-      "    'optimizer_evals': 72,\n",
-      "    'optimizer_time': 1.2052154541015625}\n"
+      "(\"OrderedDict([   ('aux_operator_eigenvalues', None),\\n\"\n",
+      " \"                ('cost_function_evals', 65),\\n\"\n",
+      " \"                ('eigenstate', {'01': 0.9921567416492215, '10': 0.125}),\\n\"\n",
+      " \"                ('eigenvalue', -1.8572750175807682),\\n\"\n",
+      " \"                (   'optimal_parameters',\\n\"\n",
+      " '                    {   ParameterVectorElement(θ[5]): 1.5683258132970883,\\n'\n",
+      " '                        ParameterVectorElement(θ[2]): 0.5470752986013949,\\n'\n",
+      " '                        ParameterVectorElement(θ[4]): -2.598325866938012,\\n'\n",
+      " '                        ParameterVectorElement(θ[7]): 0.3602071559531031,\\n'\n",
+      " '                        ParameterVectorElement(θ[3]): 6.092947713510392,\\n'\n",
+      " '                        ParameterVectorElement(θ[6]): -4.717618259450455,\\n'\n",
+      " '                        ParameterVectorElement(θ[0]): 4.296520300933687,\\n'\n",
+      " '                        ParameterVectorElement(θ[1]): 4.426962159645716}),\\n'\n",
+      " \"                (   'optimal_point',\\n\"\n",
+      " '                    array([ 4.2965203 ,  4.42696216,  0.5470753 ,  '\n",
+      " '6.09294771, -2.59832587,\\n'\n",
+      " '        1.56832581, -4.71761826,  0.36020716])),\\n'\n",
+      " \"                ('optimal_value', -1.8572750175807682),\\n\"\n",
+      " \"                ('optimizer_evals', 65),\\n\"\n",
+      " \"                ('optimizer_time', 0.25695109367370605)])\")\n"
      ]
     }
    ],
    "source": [
     "from qiskit import Aer\n",
     "\n",
-    "aqua_globals.random_seed = seed\n",
+    "algorithm_globals.random_seed = seed\n",
     "qi = QuantumInstance(Aer.get_backend('qasm_simulator'), seed_transpiler=seed, seed_simulator=seed)\n",
     "\n",
     "ansatz = TwoLocal(rotation_blocks='ry', entanglement_blocks='cz')\n",
     "slsqp = SLSQP(maxiter=1000)\n",
-    "vqe = VQE(operator=H2_op, var_form=ansatz, optimizer=slsqp, quantum_instance=qi, include_custom=True)\n",
-    "result = vqe.run()\n",
+    "vqe = VQE(ansatz, optimizer=slsqp, quantum_instance=qi, include_custom=True)\n",
+    "result = vqe.compute_minimum_eigenvalue(operator=H2_op)\n",
     "\n",
-    "pp.pprint(result)"
+    "pp.pprint(str(result))"
```

Comment:
In the text below this- again it includes values from the results that look like its needs updating

> The optimal value can be seen to be wrong too, i.e. -1.098 versus the correct -1.857.

The first value, the wrong one, looks now to be -1.074